packages/sanity/src/core/form/__telemetry__/form.telemetry.ts

```diff
@@ -0,0 +1,13 @@
+import {defineEvent} from '@sanity/telemetry'
+
+export const ExpandPortableTextInput = defineEvent({
```

Comment:
You probably based this on an existing example that was wrong, but I think it's preferred to have it in past tense, on the format "PortableTextInputExpanded"

Reply:
Makes sense!